container/root/clean.sh

```diff
@@ -1,27 +1,11 @@
 #!/bin/bash
-# IMPORTANT: This script has to be called *manually* by a child implementing this repo as a parent
+#
+########################################################################
+# Performs cleanup, ensure unnecessary packages are removed
+########################################################################
+# `apt-mark showauto` for any additional installed packages
 
-# apt-mark showauto  for any additional packages that can be cleaned up
-
-# Perform cleanup, ensure unnecessary packages are removed
-apt-get remove --purge -yq \
```

Comment:
All of this is gone because it's no longer removed at this layer...? 
Based on this comment? https://github.com/bryanlatten/docker-php/pull/96/files#r78128121